doc/whats_new/v1.5.rst

```diff
@@ -82,8 +82,8 @@ See :ref:`array_api` for more details.
 **Functions:**
 
 - :func:`sklearn.metrics.r2_score` now supports Array API compliant inputs.
-  :pr:`27904` by :user:`Eric Lindgren <elindgren>`, `Franck Charras <fcharras>`,
-  `Olivier Grisel <ogrisel>` and `Tim Head <betatim>`.
+  :pr:`27904` by :user:`Eric Lindgren <elindgren>`, :user:`Franck Charras <fcharras>`,
```

Comment:
I think we would need to remove this.

Reply:
I changed because of @ogrisel 's comment here: https://github.com/scikit-learn/scikit-learn/pull/27736#discussion_r1537951751

Reply:
Oh I see this is just adding the :user: tags. @ogrisel in that case wouldn't it be better to add this change in a separate PR so that it can be included in 1.5?

Reply:
Indeed this is a good point, we want to backport this 1.5 related fix to the 1.5.X branch but not the rest of the PR because it's a new feature/improvement that is targeted to be shipped in 1.6.